### PR TITLE
Add priority to Applied controls

### PR DESCRIPTION
**Pull request by melinoix**
Added the field priority on applied controls with colored flags on the detailView and the modeltable.

### CODE REVIEW

File: backend/core/views.py [hunk 1]
```diff
@@ -2494,7 +2502,7 @@ def export_mp_csv(request):
         "csf_function",
         "reference_control",
         "eta",
-        "effort",
+        "priority" "effort",
```
Comment (ab-smith) [inline backend/core/views.py]: ,

File: backend/core/models.py [hunk 2]
```diff
@@ -1310,6 +1310,21 @@ class Status(models.TextChoices):
...
+        ("P4", _("P4")),
+    ]
+
+    priority = models.CharField(
+        max_length=20,
+        choices=PRIORITY,
+        null=True,
+        blank=True,
+        verbose_name=_("Priority"),
+    )
```
Comment (nas-tabchiche) [inline backend/core/models.py]: This is not compliant with the specs. Priority has to be an integer, "P1", "P2"... are only labels.

**Review by nas-tabchiche — CHANGES_REQUESTED** [review]
`priority` has to be a number.
django's [PositiveSmallIntegerField](https://docs.djangoproject.com/en/5.1/ref/models/fields/#positivesmallintegerfield) seems like a good pick for this.

**Comment by eric-intuitem** [comment]
When I edit an applied control, the priority is not kept, it is set to --.

**Review by eric-intuitem — CHANGES_REQUESTED** [review]
Priority shall be correctly initialised when entering the edit page.

**Review by ab-smith — APPROVED** [review]
functional ok

**Pull request merged**